_data/misc.yaml

```diff
@@ -0,0 +1,45 @@
+releases:
+  core:
+    latest:
+      1-11: 1.11.0-RC1
+      1-10: 1.10.2
+      1-9: 1.9.3
+    all:
+      - "1.1"
+      - "1.2"
+      - "1.3"
+      - "1.4"
+      - "1.5"
+      - "1.6"
+      - "1.7"
+      - "1.8"
+      - "1.9"
+      - "1.10"
+      - "1.11"
+  plugins:
+    foreman_ansible:
+      - "0.x"
```

Comment:
What's the format of these fields?  It seems limited to only major and/or minor versions, but the Foreman releases above contains patch revisions.

I'm not sure how it'd be maintained if patch revisions _were_ added though.